src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -167,12 +176,23 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     selectPortfolioCryptoBalanceBaseUnitByFilter(s, feeAssetBalanceFilter),
   )
 
+  const amountAvailableCryptoPrecision = useMemo(() => {
```

Comment:
suggestion: last one here, this should contain runepool terminology as it is specific to it

Reply:
The fact that it's consumed by the runepool logique doesn't mean that we want to include runepool in it's name, the actual name says what it does, this is the amount available crypto precision

Reply:
Do you feel like the logic could be a problem being executed for any other pools? maybe we could include it in the memo directly?

Reply:
Fixed in [efe3439](https://github.com/shapeshift/web/pull/7449/commits/efe343902aeb8427d6dd70294cde242175ecb358)